content/pt-br/community/code-of-conduct.md

```diff
@@ -1,27 +1,30 @@
 ---
-title: Community
+title: Código de Conduta da Comunidade Kubernetes
 layout: basic
 cid: community
-css: /css/community.css
+community_styles_migrated: true
 ---
 
-<div class="community_main">
-<h1>Código de Conduta da Comunidade Kubernetes</h1>
-
+<div class="community-section" id="cncf-code-of-conduct-intro">
+<p>
 Kubernetes segue
-<a href="https://github.com/cncf/foundation/blob/master/code-of-conduct.md">CNCF Código de Conduta</a>.
+<a href="https://github.com/cncf/foundation/blob/main/code-of-conduct.md">CNCF Código de Conduta</a>.
 O texto do CNC CoC é reproduzido abaixo, a partir de
-<a href="https://github.com/cncf/foundation/blob/214585e24aab747fb85c2ea44fbf4a2442e30de6/code-of-conduct.md">commit 214585e</a>.
+<a href="https://github.com/cncf/foundation/blob/fff715fb000ba4d7422684eca1d50d80676be254/code-of-conduct.md">commit fff715fb0</a>.
 Se você perceber que isso está desatualizado, por favor
 <a href="https://github.com/kubernetes/website/issues/new">abra um bug</a>.
+</p>
+
 
-Se você notar uma violação do Código de Conduta em um evento ou meetup,
-ou em outro meio de comunicação, entre em contato com
-<a href="https://git.k8s.io/community/committee-code-of-conduct">Comitê de Código de Conduta da Comunidade Kubernetes</a>. 
-Você pode nos contatar por e-mail em<a href="mailto:conduct@kubernetes.io">conduct@kubernetes.io</a>.
+<p>
+Se você notar uma violação do Código de Conduta em um evento ou reunião, no
+Slack, ou em outro meio de comunicação, entre em contato com
+the <a href="https://git.k8s.io/community/committee-code-of-conduct">Comitê de Código de Conduta da Comunidade Kubernetes</a>. 
+Você pode nos contatar por e-mail em <a href="mailto:conduct@kubernetes.io">conduct@kubernetes.io</a>.
 Seu anonimato será protegido.
+</p>
+</div>
 
-<div class="cncf_coc_container">
+<div id="cncf-code-of-conduct">
 {{< include "/static/cncf-code-of-conduct.md" >}}
```

Comment:
It seems the content of the code of conduct itself is still out-of-date (pointing to v1.0 instead of v1.2). Could you please also update the file `static/cncf-code-of-conduct.md`?

Reply:
Hello @stormqueen1990, I added [2b7572b](https://github.com/kubernetes/website/pull/38407/commits/2b7572b51dc5cdd10d83e35aaa774a8cb93ac5a2) for this update. Let me know if it is ok. Thank you.